tdclient/job_model.py

```diff
@@ -210,14 +210,23 @@ def wait(self, timeout=None, wait_interval=5, wait_callback=None, callback=None)
         """Sleep until the job has been finished
 
         Args:
-            timeout (int): Timeout in seconds. No timeout by default.
-            wait_interval (int): wait interval in second. Default 5 seconds.
-            wait_callback (callable): A callable to be called on every tick of wait interval.
-        """
+            timeout (int, optional): Timeout in seconds. No timeout by default.
+            wait_interval (int, optional): wait interval in second. Default 5 seconds.
+            wait_callback (callable, optional): A callable to be called on every tick of
+                wait interval.
+            callback (callable, optional): [Deprecated] Same behavior as `wait_callback`
+                for compatibility.
+        """
+        if callback is not None:
+            warnings.warn(
+                "callback will be removed from future release. Please use wait_callback instaed.",
+                category=DeprecationWarning,
+            )
         started_at = time.time()
         while not self.finished():
             if timeout is None or abs(time.time() - started_at) < timeout:
                 time.sleep(wait_interval)
+                # TODO: remove `callback` argument
                 cb = wait_callback if callable(wait_callback) else callback
```

Comment:
@yyuu Seems that `callback` argument exists for the historical reason. Can we deprecate it now?

Reply:
TBQH, I don't remember much about the deprecated keyword argument. We used to have some job completion callback in some of usages, and, I have confirmed even deprecated pandas-td was implemented to use wait_callback. I think we can just go for removing it.

https://github.com/treasure-data/pandas-td/blob/master/pandas_td/td.py#L226

Reply:
thanks for your confirmation. removed d7db9a2